styles/semaphore.module.scss

```diff
@@ -0,0 +1,100 @@
+@use 'common/input.module' as input_layout;
+
+@mixin semaphore_button_mixin {
+  position: absolute;
+  //min-width: 60px;
+  //min-height: 60px;
+  display: block;
+  height: 30%;
+  width: 30%;
+
+  & > p {
+    display: none;
+  }
+
+  & > svg {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.semaphore_button_1 {
+  top: 100%;
+  left: 50%;
+  transform: translate(-50%, -100%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_2 {
+  top: 85%;
+  left: 15%;
+  transform: translate(-25%, -75%);
```

Comment:
Consider using `translate(-50%, -50%)` for all buttons and circular coordinates (static or computed) for top/left.

Reply:
FixedUp